js/script.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    "use strict"

```

Comment:
Codacy has a fix for the issue: [Replace `····"use·strict"␍` with `··"use·strict";`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456242)
```suggestion
  "use strict";
```